terminologies/terminology/metaDataModelType.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/terminology/metaDataModelType",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Terminology",
+  "name": "metadata model type",

```

Comment:
(meta)data model type

Reply:
to explain: it covers both metadata model types (e.g. openMINDS) AND data model types (e.g. BIDS)

Reply:
That's what I thought but didn't trust myself :D